modules/persistent-storage-csi-azure-file-cross-sub-dynamic-pre-provisioning-pv-pvc-procedure.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent_storage-csi-azure-file.adoc
+//
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-azure-file-cross-sub-dynamic-pre-provisioning-pv-pvc-procedure_{context}"]
+= Static provisioning across subscriptions for Azure File by creating a PV and PVC:
+
+.Prerequisites
+* Installed {product-title} cluster on Azure with the service principal or managed identity as an Azure identity in one subscription (call it Subscription A)
+
+* Access to another subscription (call it Subscription B) with the storage that is in the same tenant as the cluster
+
+* Logged in to the Azure CLI
+
+.Procedure
+. For your Azure File share, record the resource group, storage account, storage account key, and Azure File name. These values are used for the next steps.
+
+. Create a secret for the persistent volume parameter `spec.csi.nodeStageSecretRef.name` by running the following command:
++
+[source, terminal]
+----
+$ oc create secret generic azure-storage-account-{storageaccount-name}-secret --from-literal=azurestorageaccountname="<azure-storage-account-name>" --from-literal azurestorageaccountkey="<azure-storage-account-key>" --type=Opaque
```

Comment:
we have a mix of <> and {} in that command
we also have {storageaccount-name} and <azure-storage-account-name> which are the same thing IIUC
if so let's keep <azure-storage-account-name> for both

Reply:
I've changed to <> since that's what we generally use.